From: Jiri Kosina <jkosina@suse.cz>
Subject: Input: add Gericom Bellagio to nomux blacklist
References: bnc#404892

Gericom Bellagio needs to be added to nomux blacklist, otherwise its
touchpad misbehaves.

Reported-by: Roland Kletzing <roland.kletzing@materna.de>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>

---
 drivers/input/serio/i8042-x86ia64io.h |    7 +++++++
 1 file changed, 7 insertions(+)

--- linux-2.6.25-SL110_BRANCH.orig/drivers/input/serio/i8042-x86ia64io.h
+++ linux-2.6.25-SL110_BRANCH/drivers/input/serio/i8042-x86ia64io.h
@@ -305,6 +305,13 @@ static struct dmi_system_id __initdata i
 			DMI_MATCH(DMI_PRODUCT_NAME, "Aspire 1360"),
 		},
 	},
+	{
+		.ident = "Gericom Bellagio",
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Gericom"),
+			DMI_MATCH(DMI_PRODUCT_NAME, "N34AS6"),
+		},
+	},
 	{ }
 };
 
